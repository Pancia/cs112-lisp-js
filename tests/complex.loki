(def f (fn [] true)) (print (f))
(def f (fn [] (+ 2 3))) (print (f))
(def f (fn [x] (+ x 1))) (print (f 9))
