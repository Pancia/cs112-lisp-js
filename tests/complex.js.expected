true
5
10
